test_apps/configs/README.md

```diff
@@ -19,4 +19,4 @@ Example wolfSSL configuration file templates for use when autoconf is not availa
 To use these with autoconf:
 
 1. Copy file to root as `user_settings.h`.
-2. Run `./configure --enable-usersettings --disable-examples && make`
+2. Run `./configure --enable-usersettings --disable-test_apps && make`
```

Comment:
If we are changing the configure option I don't see any of the changes in `configure.ac`. I'm not sure if we want to change that or not....it may break peoples build when they upgrade.

Reply:
Agree, that should not change and doc find/replace needs to be reverted.

Reply:
All docs have been reverted and changed so that only directories are modified

Reply:
*directories and file paths